github/rubocop-github
# Update codeowners, dependencies and build config to fix Dependabot PRs

- ran `bundle update`
- Add Ruby 3.2 to test matrix
- Add .ruby-version
- Update Bundler version and include `ruby` platform in lockfile
- Ensure Rubocop::Config is passed to test helper
- Update codeowners to `rubocop-reviewers` team

~~I didn't update any of the actual dependencies (e.g. rubocop, minitest, activesupport, etc.)~~

I _did_ update the dependencies. Rubocop now returns all of the new cops that _could_ be enabled/disabled. I think this is fine to leave as-is because updating the Gemfile.lock in this gem won't affect any applications that bundle this gem (I didn't update any of the gemspec constraints).
